libs/dissect/src/main/java/org/elasticsearch/dissect/DissectParser.java

```diff
@@ -296,6 +296,26 @@ public Map<String, String> forceParse(String inputString) {
         return results;
     }
 
+    public List<String> outputKeyNames() {
```

Comment:
Nit: `outputKeyNames()` versus `outputKeys()`? Is there something *truly* *name*-y about these? Or are they just keys?

Reply:
👍 outputKeys() is just fine I guess